release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -156,41 +156,41 @@ In the following tables, features are marked with the following statuses:
 
 |====
 
-[discrete]
-=== Monitoring deprecated and removed features
+//[discrete]
+//=== Monitoring deprecated and removed features
```

Comment:
I do not see this section in the [4.13 release notes](https://docs.openshift.com/container-platform/4.13/release_notes/ocp-4-13-release-notes.html). Maybe a reason existed for this?

Reply:
Yeah, I think we can take these out if we don't hear anything closer to GA.